pkg/crc/machine/start.go

```diff
@@ -137,19 +132,27 @@ func growRootFileSystem(sshRunner *crcssh.Runner, preset crcPreset.Preset) error
 	if _, _, err := sshRunner.RunPrivileged("Remounting /sysroot read/write", "mount -o remount,rw /sysroot"); err != nil {
 		return err
 	}
-	if _, _, err = sshRunner.RunPrivileged(fmt.Sprintf("Growing %s filesystem", rootPart), "xfs_growfs", rootPart); err != nil {
+	if _, _, err = sshRunner.RunPrivileged(fmt.Sprintf("Growing %s filesystem", rootPart), "xfs_growfs", "/sysroot"); err != nil {
```

Comment:
Previous comments are regarding "`/sysroot` which seems to be a shared mount. seeing the change looks awkward without ulling this out as a more meaningful constant, like `rootMount`

Reply:
Updated with `rootFS` variable.